packages/token-lists/src/top-100.ts

```diff
@@ -0,0 +1,49 @@
+import fs from "fs";
+import path from "path";
+import { request, gql } from 'graphql-request'
+import { getAddress } from "@ethersproject/address"
+
+interface TokenEntity {
+  id: string;
+  name: symbol;
+  symbol: string;
+  decimals: string;
+}
+
+const getTokens = async () => {
+    const { tokens } = await request(
+        "?_?",
```

Comment:
What is this?

Reply:
That was not to leak any non-public endpoint, but got another way to do it; update in latest commit.